src/cljs/rems/administration/create_catalogue_item.cljs

```diff
@@ -188,13 +188,34 @@
          :item-selected? item-selected?
          :on-change #(rf/dispatch [::set-selected-workflow %])}])]))
 
+(defn resourse-dropdown [r language counts]
+  (let [organisation? (get-in r [:organization :organization/short-name (keyword language)])
+        duplicate? (> (get counts (:resid r)) 1)
+        licenses? (not-empty (:licenses r))]
+    (str (:resid r)
+      ;; when organisation is found, show organisation
+         (when organisation?
+           (str/join " " [" (org:" organisation? ")"]))
+      ;; when duplicate is found, show duplicate
+         (when duplicate?
+         ;; when licenses are found, show licences, but only if it is duplicate
+           (when licenses?
+             (str/join " "
+                       [" (licenses:"
+                        (str/join ", " (mapv
+                                        (fn [l] (:title ((keyword language) (:localizations l))))
```

Comment:
consider: `(fn [l] (get-in l [:localizations language :title]))`

Reply:
That could be a better way to do it, thanks!